docs/howto/craft-a-snap/index.rst

```diff
@@ -1,23 +1,20 @@
 .. _craft-a-snap:
 
-Craft for platforms
-===================
+Craft a snap
+============
 
-Apps are often built using programming languages and technologies that
-are supplied with their own build tools. Snapcraft uses plugins to integrate
-with these tools in order to create snaps.
+Snapcraft can package apps and programs from multiple languages, frameworks,
+and meta build systems. You can create snaps from apps you've already built and
+zipped, or directly from source. No matter their underlying code, all apps gain
+the same benefits from being packed into snaps.
 
-Snapcraft can package apps and programs from many programming languages,
-frameworks, and meta build systems, which for simplicity we call *platforms*
-(no relation to the term as it's used in snap recipes).
+Snapcraft uses special :ref:`plugins` to hook into and manage the different
+languages and frameworks, so snap crafting is usually separated between the
+different plugins.
```

Comment:
```suggestion
Snapcraft uses special :ref:`plugins` to hook into and manage the different
languages, and provides :ref`extensions` to support certain frameworks. So
snap crafting is usually separated between the different plugins.
```

I am not sure we have a ref for extensions, but ros support and gnome are supported through extensions for them (we can keep it unrefed if we do not have a landing page for it in rtd)